core/console/src/main/java/org/eclipse/rdf4j/console/Federate.java

```diff
@@ -77,35 +77,33 @@ private void federate(boolean distinct, boolean readonly, String fedID, Deque<St
 		if (LOGGER.isDebugEnabled()) {
```

Comment:
This is where the bug occurred, this should never have had an else clause linked to it, as that linked it to control flow.